src/mechaclient.py

```diff
@@ -102,6 +102,10 @@ def __init__(self, *args, mecha_config=None, **kwargs):
         try:
             self._last_user_message[user.casefold()] = sanitized_message  # Store sanitized message
             ctx = await Context.from_message(self, channel, user, sanitized_message)
+            if not ctx.words:
+                logger.trace("ignoring empty message")
```

Comment:
[Suggestion] As this came up as a bug, should we add a regression test?

Reply:
agreed.